### ISSUE TITLE
Select list option with dynamically added data

### ISSUE THREAD

**Issue by abhishekkherde** [Xarksass/BootstrapSelect#1]
I am trying to added data dynamically in "UL LI" structure using ajax, when trying to select the dynamically added "li" is not getting selected. Thanks in Advance..

**Comment by Xarksass** [comment]
It depends on how and when you instantiate the select : 
If you use the selectpicker class, it will instantiate the select right after the page has been loaded, so it will work with the li items which were in the list at the instantiation. Same thing if you use the  bootstrapSelect() function just after the loading of the page.

So it depends on what you want to do:
- If you simply add some items in the list only once with ajax, then you just have to instantiate the select (with bootstrapSelect() ) right after the items have been added.
- If you're gonna change the items in the list multiple times (add or remove), you're gonna have to manually "destroy" the instance of the plugin and re-instantiate a new one right after.

I didn't thought about a use with ajax when I've developed this plugin, but it would be a plus of taking this case into account. I will work on it ;)

ps: Sorry for this late response, I was on holiday so I've check my emails only now.
